robottelo/hosts.py

```diff
@@ -1187,6 +1187,24 @@ def register_to_dogfood(self, ak_type='satellite'):
                 f'Error during registration, command output: {cmd_result.stdout}'
             )
 
+    def register_to_cdn(self):
+        """Subscribe satellite to CDN"""
+        self.remove_katello_ca()
+        major_version = self.os_version.major
+        release_version = f'{major_version}Server' if major_version < 8 else f'{major_version}'
```

Comment:
Shouldn't subscription-manager be able to set it itself while registering? Why be explicit ?

Reply:
As I referred to the existing subscribe_satellite fixture for this helper func, I'm not sure why it is written to include this explicitly and whether it should be kept or not